src/migrations/20191218075931-create-users.js

```diff
@@ -64,9 +64,6 @@ export function up(queryInterface, Sequelize) {
       type: Sequelize.BOOLEAN,
       defaultValue: false
     },
-    lineManagerId: {
```

Comment:
Why is this column removed?

Reply:
@adesege It was removed because it was a duplicate of `line 51`. After rebasing i noticed that there is someone who edited `user migrations && models` by adding `lineManagerId` while before it was created at `line 51` in this file.